src/api/data-types.md

```diff
@@ -33,12 +33,12 @@ typically represented as an [ISO 4217][]{:.external} code.
 
 {% h4 Fields %}
 
-| Name     | Type               | Description                                                |
-|----------|--------------------|------------------------------------------------------------|
+|   Name   |        Type        |                        Description                         |
+| -------- | ------------------ | ---------------------------------------------------------- |
 | amount   | {% dt BigNumber %} | Value in the currency's smallest denomination (eg. cents). |
 | currency | String             | Currency code (eg. "NZD").                                 |
 
-
+See [Asset Types][] for the list of possible `currency` values for each Asset Type.
```

Comment:
I put it in there because there is a link to Monetary in v2 payment request model. If i put this at the bottom of the PaymentRequest model then it won't be that obvious to the user where the currency variable actually goes. Is there a better spot for it?